Cargo.toml

```diff
@@ -24,3 +24,4 @@ rand = "0.7"
 reqwest = { version = "0.11", features = ["json", "blocking", "gzip"] }
 askama = "0.12"
 bytes = "0.4"
+natord = "1.0.9"
```

Comment:
I'm still testing this... but could you please change the `version` above to `1.3.3`?